Board.java

```diff
@@ -0,0 +1,222 @@
+import java.util.HashSet;
+import java.util.Scanner;  
+import java.util.Set;
+
+class Board{ 
+    
+    public class Point{
+        int x, y;
+        Point(int x, int y){
+            this.x = x;
+            this.y = y;
+        }
+        
+        @Override
+        public String toString(){
+            return "["+x+", "+y+"]";
+        }
+        
+        @Override
+        public boolean equals(Object o){
+            return o.hashCode()==this.hashCode();
+        }
+
+        @Override
+        public int hashCode() {
+            return Integer.parseInt(x+""+y);
+        }
+    }
+    
+    private final char[][] board;
+    int WScore, BScore, remaining;
+    private final char boardX[] = new char[]{'A','B','C','D','E','F','G','H'};
+    
+    public Board(){ 
+        board = new char[][]{
+            {'_','_','_','_','_','_','_','_',},
+            {'_','_','_','_','_','_','_','_',},
+            {'_','_','_','_','_','_','_','_',},
+            {'_','_','_','W','B','_','_','_',},
+            {'_','_','_','B','W','_','_','_',},
+            {'_','_','_','_','_','_','_','_',},
+            {'_','_','_','_','_','_','_','_',},
+            {'_','_','_','_','_','_','_','_',},
+        };  
+    }
+    
+    private void findPlaceableLocations(char player, char opponent, HashSet<Point> placeablePositions){ 
+        for(int i=0;i<8;++i){
```

Comment:
never write direct numbers in code. Use n as a parameter and assign n=8 in the init method. Use n for all purposes so that it can easily be changed to serve a bigger board. Othello, as far as I know, can be played on as big a board as you want.